src/components/book-input-decorator.js

```diff
@@ -80,6 +80,11 @@ class BookInputDecorator extends LitElement {
       <slot name="button"></slot>
     `;
   }
+  static get properties() {
```

Comment:
Can you add a blank line before `static get properties()` so more consistent with the other files?

Reply:
done.

Reply:
@salah-saleh Thanks for the PR!